docker-compose.yml

```diff
@@ -1,7 +1,6 @@
-version: '3'
 services:
   app:
-    image: cloudcodeai/kaizen-app:latest
+    image: cloudcodeai/kaizen-app:v0.4.9
```

Comment:
keep it latest or else we need to update this everytime

Reply:
"latest" throws warning errors, so probably we have to change it every time.